v2/internal/controllers/storage_storageaccount_crud_20210401_test.go

```diff
@@ -131,6 +134,9 @@ func StorageAccount_QueueServices_Queue_20210401_CRUD(tc *testcommon.KubePerTest
 		},
 	}
 
+	// Don't try to delete directly, this is not a real resource - to delete it you must delete its parent
+	tc.AddAnnotation(&queue.ObjectMeta, "serviceoperator.azure.com/reconcile-policy", "detach-on-delete")
```

Comment:
This doesn't seem right to me? We can delete queues for sure. We do just below in fact (see the defer)